Health/Presentation/Dashboard/DashboardViewController.swift

```diff
@@ -376,8 +376,9 @@ fileprivate extension DashboardViewController {
                 message: """
                          대시보드 현황을 공유하려면 걸음 수 건강 데이터 접근 권한이 필요합니다.
                          
-                         경로: 프로필(우측 상단) ⏵ 개인정보 보호 ⏵ 앱 ⏵ Health  
-                         위 경로에서 앱의 데이터 접근 권한을 해제하거나 다시 활성화할 수 있습니다.
+                         아래 경로에서 앱의 데이터 접근 권한을 해제하거나 다시 활성화할 수 있습니다.
```

Comment:
온보딩이랑 맞추면 좋을 것 같은데 어떠신가요?

```suggestion
                         아래 경로에서 앱의 건강 데이터 접근 권한을 해제하거나 다시 활성화할 수 있습니다.
```